package.json

```diff
@@ -73,6 +73,6 @@
   },
   "optionalDependencies": {
     "mysql": "^2.11.1",
-    "sqlite3": "^4.0.0"
+    "sqlite3": "^4.0.4"
```

Comment:
Why min version 4.0.4?

Reply:
👋 No special reason~

Reply:
I'd suggest to use the min version which officially supports node 10?

Reply:
Yes, you are right. I didn't consider it thoughtfully. 😀

Reply:
👻 Thanks for your help!!